### PR TITLE
etcdctl/ctlv3/command: add use time for defrag command

### PR DESCRIPTION
**Pull request by qsyqian**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: etcdctl/ctlv3/command/defrag_command.go [hunk 1]
```diff
@@ -53,13 +54,15 @@ func defragCommandFunc(cmd *cobra.Command, args []string) {
 	c := mustClientFromCmd(cmd)
 	for _, ep := range endpointsFromCluster(cmd) {
 		ctx, cancel := commandCtx(cmd)
+		start := time.Now()
 		_, err := c.Defragment(ctx, ep)
+		d := time.Now().Sub(start)
 		cancel()
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to defragment etcd member[%s] (%v)\n", ep, err)
+			fmt.Fprintf(os.Stderr, "Failed to defragment etcd member[%s]. use %s. (%v)\n", ep, d.String(), err)
```
Comment (hexfusion) [inline etcdctl/ctlv3/command/defrag_command.go:62]: nit:  s/use/took?

Comment (qsyqian) [reply etcdctl/ctlv3/command/defrag_command.go]: thanks for review

**Review by hexfusion — APPROVED** [review]

**Comment by hexfusion** [comment]
In general, this value will probably equal your client timeout but having that observability could help triage "why did defrag fail".

**Comment by qsyqian** [comment]
> your

yes, you are right. another reason is that i want to know how much time it use when i execute etcdctl defrag. without this value i have to take a look at the sever log.

**Review by spzala — APPROVED** [review]
lgtm
Thanks @qsyqian

**Pull request merged**